### ISSUE TITLE
Critère 1.2 : placement de l'attribut aria-hidden="true"

### ISSUE THREAD

**Issue by inseo** [accessibility-luxembourg/ReferentielAccessibiliteWeb#1]
Dans la suite de tests du critère 1.2, il est explicitement demandé à ce que l'attribut `aria-hidden="true"` soit placé sur la balise image. Or, cet attribut étant héritable, si le parent de l'image en est porteur, l'image hérite de la valeur de cet attribut de manière implicite.

Si l'on applique le [test 1.2.4](https://accessibilite.public.lu/fr/raweb1/criteres.html#test-1-2-4) à lettre, le code ci-après devrait être considéré comme Non Conforme, sans pour autant que cela ne pose de problème d'accessibilité :
```
<span aria-hidden="true">
  <svg width="6" height="10" viewBox="0 0 6 10">
    <path d="M2.21932 4.99999L5.51932 8.29999L4.57665 9.24266L0.333984 4.99999L4.57665 0.757324L5.51932 1.69999L2.21932 4.99999Z" />
  </svg>
</span>
```

De fait, il pourrait être utile de revoir la méthodologie de test pour valider les cas où les balises image héritent implicitement de la valeur `aria-hidden` de leur parent.

**Comment by AlainVagner** [comment]
Bonjour @inseo, merci pour le retour. Effectivement ce cas n'est pas géré actuellement.
Est-ce qu'une telle formulation pourrait convenir ?
`La balise <svg>, ou l'un de ses parents, possède un attribut WAI-ARIA aria-hidden="true".`

Ce changement devrait être réalisé dans tous les critères et tests qui impliquent l'attribut aria-hidden="true".

**Comment by luce-carevic** [comment]
Il n'y a pas actuellement de problème sur ce cas avec le RGAA ou le RAWeb lors d'un audit. 

Si un élément est contenu dans un parent masqué avec aria-hidden="true", il « n'existe plus » et n'est donc pas pris en compte. Dans le cas remonté, le critère ne s'applique pas car il n'y a plus de svg sur lequel l'appliquer.

La précision reste néanmoins peut-être utile pour qui ne connaîtrait pas le principe de fonctionnement de aria-hidden="true", mais il n'y a pas de blocage en l'état.

**Comment by AlainVagner** [comment]
Effectivement, c'est plus une question de formulation dans le référentiel. Si le référentiel dit "La balise <svg> possède un attribut WAI-ARIA aria-hidden=true", dans le cas de l'exemple donné, ce n'est pas le cas. Ce que l'on cherche à savoir c'est si cet élément svg est globalement affecté par un aria-hidden=true, qu'il soit présent sur l'élément lui-même ou l'un de ses parents. 
Je serais donc pour modifier la formulation a minima pour refléter ce mode de fonctionnement comme proposé ci-dessus.

**Comment by AlainVagner** [comment]
Pour des raisons de clarté, et pour éviter des changements trop invasifs dans le référentiel, nous proposons de centraliser la modification dans une note de glossaire dans : image de décoration.
Nous proposons d'ajouter la note suivante : 

> Si une image se trouve imbriquée dans une balise possédant un attribut `aria-hidden="true"`, alors elle est considérée comme décorative.

Qu'en pensez-vous ?

**Comment by AlainVagner** [comment]
Traité dans le RAWeb 1.1
